values.yaml

```
@@ -1,26 +1,21 @@
-ingress:
-    enabled: true
-    host: ui.public.url
-    webdavHost: webdav.public.url
-    className: nginx-public
-    annotations:
-        cert-manager.io/cluster-issuer: letsencrypt-http01
-        nginx.ingress.kubernetes.io/rewrite-target: /
-        nginx.ingress.kubernetes.io/proxy-read-timeout: "3600"
-        nginx.ingress.kubernetes.io/proxy-send-timeout: "3600"
-        nginx.ingress.kubernetes.io/proxy-body-size: 512m
-        nginx.ingress.kubernetes.io/server-snippets: '
-            location / {
-                proxysetheader Upgrade $httpupgrade;
-                proxyhttpversion 1.1;
-                proxysetheader X-Forwarded-Host $httphost;
-                proxysetheader X-Forwarded-Proto $scheme;
-                proxysetheader X-Forwarded-For $remoteaddr;
-                proxysetheader Host $host;
-                proxysetheader Connection "upgrade";
-                proxycachebypass $httpupgrade;
-            }'
-    tls: true
+voltaserve:
+    ingress:
+        ui:
+            enabled: true
+            host: ui.public.url
+            annotations:
+                {}
+            className: nginx-public
+            tls:
+                {}
+        webdav:
+            enabled: true
+            annotations:
+                {}
+            host: webdav.public.url
+            className: nginx-public
+            tls:
+                {}
```

```suggestion
                []
        webdav:
            enabled: true
            annotations:
                {}
            host: webdav.public.url
            className: nginx-public
            tls:
                []
```
I'm curious why the linter doesn't care

---

idk if both of those values do not represent null

---

Well, I'm wondering if it can be overridden as array if it's defined as object here. We might want to lint with common customization overrides